Student/Student/People/PeopleListPresenter.swift

```diff
@@ -47,7 +47,7 @@ class PeopleListPresenter {
 
     func viewIsReady() {
         colors.refresh()
-        users.refresh()
+        users.exhaust(while: { _ in true })
```

Comment:
The `PeopleListViewController` handles getting the next page on scroll so we don't want to exhaust here.

https://github.com/instructure/canvas-ios/blob/77a00f03b3de14d04c0eb04121e099c89f06caa3/Student/Student/People/PeopleListViewController.swift#L93-L99

Reply:
ahh ok. I noticed a few presenters that were refreshing when they should have been exhausting. Didn't think about the view controller handling pagination. I'll change this back.